### PR TITLE
Chore/99 update dbdocs

### PR DESCRIPTION
**Pull request by kmcelwee**
Nothing really fancy here, just following documentation from https://dbdocs.io/guide, section 8. The build was successful: https://dbdocs.io/princetoncdh/geniza

### CODE REVIEW

File: .github/workflows/dbdocs.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ env:
 
 jobs:
   python-unit:
-    name: Python unit tests
+    name: Publish dbdocs
```
Comment (thatbudakguy) [inline .github/workflows/dbdocs.yml:20]: can we change the label here too? `python-unit` seems wrong

File: .github/workflows/dbdocs.yml [hunk 2]
```diff
@@ -28,7 +28,9 @@ jobs:
         with:
           node-version: ${{ env.NODE_VERSION }}
       - name: Install dbdocs
-        run: npm install dbdocs
+        run: sudo npm install -g dbdocs
```
Comment (thatbudakguy) [inline .github/workflows/dbdocs.yml:31]: `sudo` is normally discouraged when using `npm` (for the same reason you never `sudo pip install ...`). does dbdocs require this?

Comment (thatbudakguy) [reply .github/workflows/dbdocs.yml:31]: just saw that they say it "may be required". can we try without? it's a CI runner so maybe not too much impact of `sudo install`ing it, but I'd rather not if we don't have to

Comment (kmcelwee) [reply .github/workflows/dbdocs.yml]: It was still successful after removing `sudo`. I also updated the label. Good to merge?

**Review by thatbudakguy — APPROVED** [review]
perfect, let's merge it!

**Pull request merged**